src/ophys_etl/pipelines/suite2p_registration.py

```diff
@@ -201,20 +294,21 @@ def run(self):
                 pilim.save(dst_path)
             self.logger.info(f"wrote {dst_path}")
 
-        # Suite2P ops file contains at least the following keys:
-        # ["Lx", "Ly", "nframes", "xrange", "yrange", "xoff", "yoff",
-        #  "corrXY", "meanImg"]
-        ops = np.load(ops_path, allow_pickle=True)
-
         # Save motion offset data to a csv file
         # TODO: This *.csv file is being created to maintain compatability
         # with current ophys processing pipeline. In the future this output
         # should be removed and a better data storage format used.
         # 01/25/2021 - NJM
+        x_is_clipped = np.zeros(delta_x.size).astype(bool)
+        x_is_clipped[x_clipped] = True
+        y_is_clipped = np.zeros(delta_y.size).astype(bool)
+        y_is_clipped[y_clipped] = True
         motion_offset_df = pd.DataFrame({
             "framenumber": list(range(ops.item()["nframes"])),
-            "x": ops.item()["xoff"],
-            "y": ops.item()["yoff"],
+            "x": delta_x,
+            "y": delta_y,
+            "x_clipped": x_is_clipped,
```

Comment:
I thought these columns were going to store the unclipped values? Did that turn out to be too memory inefficient?

Reply:
how about I call it, for example, `x_pre_clip` and just dump `ops.item()['xoff']. ? If someone really cares, they can compare the columns.

Reply:
changed it